collection-scripts/gather_odf_client

```diff
@@ -0,0 +1,72 @@
+#!/usr/bin/env bash
```

Comment:
I believe you are missing storageclassclaims

Reply:
Storageclassclaims are fetched as a clusterscopedresources. The storageclassclaims created with storageclient are also captured in must-gather

Reply:
ack, then I don't see them as part of this PR and I highly doubt they are being collected already.